CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 
 ## Bug Fixes
 
+- [#338](https://github.com/pybop-team/PyBOP/pull/338) - Fixes GaussianLogLikelihood class, adds integration tests, updates non-bounded parameter implementation and bugfix to CMAES construction.
```

Comment:
Please clarify "updates non-bounded parameter implementation", perhaps: applies default bounds to parameters with priors?